mlir/python/CMakeLists.txt

```diff
@@ -3,6 +3,8 @@ include(AddMLIRPython)
 # TODO: Add an upstream cmake param for this vs having a global here.
 add_compile_definitions("MLIR_PYTHON_PACKAGE_PREFIX=mlir_quantum.")
 
+# Ignore nanobind warnings
+add_compile_options(-w)
```

Comment:
Out of curiosity, is this what Lee recommended in the end? I thought he had a somewhat different trick.

Separately though, I would just like to ensure a blind warning disable like this is very targeted. For instance with Stim, I used the following:
```
target_compile_options(libstim PRIVATE -w)
```
Are we sure the line above doesn't affect code we'd like to have warnings on?